src/pages/ScoreReport.vue

```diff
@@ -839,6 +840,17 @@ const exportSelected = (selectedRows) => {
         ? _get(assessment, 'scores.computed.composite')
         : _get(assessment, `scores.computed.composite.${rawScoreKey}`);
       tableRow[`${taskDisplayNames[taskId]?.name ?? taskId} - Support Level`] = support_level;
+      if (assessment.reliable !== undefined && !assessment.reliable && assessment.engagementFlags !== undefined) {
+        const reliabilityKeys = Object.keys(assessment.engagementFlags);
+        if (reliabilityKeys.length > 0) {
+          const reliabilityString = reliabilityKeys.map((key) => _lowerCase(key)).join(', ');
```

Comment:
```suggestion
          const reliabilityString = reliabilityKeys.map((key) => _capitalize(key)).join(', ');
```
Not a critical change, but I wonder if we should capitalize these using the lodash util to capitalize the first letter of a string? Or write a function to capitalize each word to match the rest of the document. The all-lowercase reliability flags look award against evereything else, in my opinion.

![image](https://github.com/yeatmanlab/roar-dashboard/assets/100953047/46682c5c-ca6a-4db3-ae70-a6ffd511d00f)

Reply:
Cool, let me try this

Reply:
I think _lowerCase is better in this case, because otherwise it will export like this: ResponseTimeTooFast, AccuracyTooLow and not as separate words

Reply:
I should actually have written this:

`const reliabilityString = reliabilityKeys.map((key) => _capitalize(_lowerCase(key))).join(', ');`

Which gives this result, so at least the first letter of each key is capitalized:

![image](https://github.com/yeatmanlab/roar-dashboard/assets/100953047/b328f476-ebdd-41cd-ab67-3ce123145466)